examples/java-basic/proto/example/v1/person.proto

```
@@ -0,0 +1,29 @@
+  int32 id = 1;
+  string name = 2;
+  string email = 3;
+  repeated PhoneNumber phones = 4;
+  
+  message PhoneNumber {
+    string number = 1;
+    PhoneType type = 2;
+  }
+  
+  enum PhoneType {
+    MOBILE = 0;
```

Buf (1.54.0)

Enum value name "MOBILE" should be prefixed with "PHONE_TYPE_".

(ENUM_VALUE_PREFIX)